### PR TITLE
feat(j-s): Defender Scheduled Case Overview

### PR DESCRIPTION
**Pull request by gudjong**
# Defender Scheduled Case Overview

https://app.asana.com/0/1199153462262248/1202086011710985/f

## What

- Adds a scheduled case overview page for defenders.
- Stop sending the request as an attachment in email to the defender, replacing it with a link to RVG.

## Why

Remove the need for sending documents to defenders with email.

## Screenshots / Gifs

image

Email to defender with link:
image

Email to defender without link:
image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web-e2e/src/integration/Defender/caseOverview.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      id: 'test_id',
+      created: '2020-09-16T19:50:08.033Z',
+      modified: '2020-09-16T19:51:39.466Z',
+      state: CaseState.ACCEPTED,
+      origin: CaseOrigin.RVG,
+      type: CaseType.CUSTODY,
+      policeCaseNumber: '007-2021-202000',
+    }
+
+    cy.stubAPIResponses()
```
Comment (oddsson) [inline apps/judicial-system/web-e2e/src/integration/Defender/caseOverview.ts:27]: You are calling stubAPIResponses twice. It's only needed in the beforeEach

File: apps/judicial-system/web/messages/Core/defenderCaseOverview.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+// Strings on signed verdict overview screen
+export const defenderCaseOverview = {
+  title: defineMessages({
+    investigationCaseRejected: defineMessage({
+      id:
+        'judicial.system.core:defender_case_overview.title.investigation_case_rejected',
+      defaultMessage: 'Kröfu um rannsóknarheimid hafnað',
+      description:
+        'Notaður sem titil á yfirlitsskjá verjanda þegar kr0fu um rannsóknarheimild hefur verið hafnað.',
```
Comment (oddsson) [inline apps/judicial-system/web/messages/Core/defenderCaseOverview.ts:11]:
```suggestion
        'Notaður sem titil á yfirlitsskjá verjanda þegar kröfu um rannsóknarheimild hefur verið hafnað.',
```

File: apps/judicial-system/web/messages/Core/defenderCaseOverview.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    caseDismissed: defineMessage({
+      id: 'judicial.system.core:defender_case_overview.title.case_dismissed',
+      defaultMessage: 'Kröfu vísað frá',
+      description:
+        'Notaður sem titill á yfirlitsskjá verjanda þegar kröfu hefur verið vísað frá.',
+    }),
+    investigationCaseAccepted: defineMessage({
+      id:
+        'judicial.system.core:defender_case_overview.title.investigation_case_accepted',
+      defaultMessage: 'Krafa um rannsóknarheimid samþykkt',
```
Comment (oddsson) [inline apps/judicial-system/web/messages/Core/defenderCaseOverview.ts:29]:
```suggestion
      defaultMessage: 'Krafa um rannsóknarheimild samþykkt',
```

**Review by oddsson — CHANGES_REQUESTED** [review]
There are some typos that need fixing

File: apps/judicial-system/web-e2e/src/integration/Defender/caseOverview.ts (hunk 1 above)
Comment (gudjong) [reply apps/judicial-system/web-e2e/src/integration/Defender/caseOverview.ts:27]: You are right, will fix. I noticed this is done in a lot of e2e tests though.

File: apps/judicial-system/web/messages/Core/defenderCaseOverview.ts [hunk 4]
```diff
@@ -0,0 +1,88 @@
+import { defineMessage, defineMessages } from 'react-intl'
+
+// Strings on signed verdict overview screen
+export const defenderCaseOverview = {
+  title: defineMessages({
+    investigationCaseRejected: defineMessage({
+      id:
+        'judicial.system.core:defender_case_overview.title.investigation_case_rejected',
+      defaultMessage: 'Kröfu um rannsóknarheimid hafnað',
```
[the remark this answers is not in the document]
Comment (gudjong) [reply apps/judicial-system/web/messages/Core/defenderCaseOverview.ts:9]: Good catch. This is a copy paste error and is also found in an existing Contentful string. I will fix both the code and Contentful.

**Review by oddsson — APPROVED** [review]

**Pull request merged**